files/image_config/monit/process_checker

```diff
@@ -51,7 +51,7 @@ def check_process_existence(container_name, process_cmdline):
 
             for process in psutil.process_iter(["cmdline", "status", "pid"]):
                 try:
-                    if ((' '.join(process.cmdline())).startswith(process_cmdline) and process.status() in ["running", "sleeping"]):
```

Comment:
@FengPan-Frank one problem i see is that if the process is continuously in disk-sleep then its not expected and monit should raise alert. So, I would suggest, to remove the print from process_checker for process not running and let the monit check the return status of process_checker and raise the alert:-

```
check program pmon|ledd with path "/usr/bin/process_checker pmon /usr/bin/python /usr/bin/ledd"
    if status != 0 for 5 times within 5 cycles then alert repeat every 1 cycles
```

This way, we get alert if the process is in disk-sleep for 5 * 30 secs...

Reply:
Hi @prgeor , refer current logic https://github.com/sonic-net/sonic-buildimage/blob/36e65035bab6c0356ebcb9d0b1851553c9b5901c/dockers/docker-platform-monitor/base_image_files/monit_pmon.j2#L45C38-L45C38, all processes in monit will get alert if 5 consecutive times within 5 Monit cycles (1 minute per cycle)

Are you suggesting we adjust monit logic for sensord particularly?

Reply:
@FengPan-Frank remove this print https://github.com/sonic-net/sonic-buildimage/blob/8cf5b6827c8a1204ecd5ad0074cfb682d31bbd8e/files/image_config/monit/process_checker#L78

and let the monit alert print the syslog

Reply:
@prgeor Are you suggesting remove process_checker logging and when checking process status, do NOT skip "disk-sleep", as well as "sleeping" considering keep in "sleeping" state may also be somehow problematic, right?

Reply:
@FengPan-Frank for now I would suggest to keep the behavior same for 201911 except remove the extra print() and let monit watch over.

Reply:
@FengPan-Frank please capture the monit log which you see in your DUT in the PR description. I got confused, because of the printf()